src/Microsoft.ML.Experimental/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+using Microsoft.ML;
+
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Tests" + PublicKey.TestValue)]
```

Comment:
Instead of doing this, we should change the test to use the public APIs. Basically `new DatabaseLoader` => `mlContext.Data.CreateDatabaseLoader`

Reply:
Fixed.

Reply:
Do we still need this?

---
In reply to: [308308500](https://github.com/dotnet/machinelearning/pull/4035#discussion_r308308500) [](ancestors = 308308500)

Reply:
I think this is the only comment I have left on this PR. If this is resolved, and all the builds pass, I think we will be ready to merge this.

Reply:
Yes. (sorry forgot to update the comment)

This is still needed for converting from the DbType to the InternalKind to the public DataKind.